### ISSUE TITLE
Parameter fetching is broken

### ISSUE THREAD

**Issue by remram44** [ViDA-NYU/reprozip#340]
Broken at least on Python 3. Not sure if this is because of a lib update or a change in code.

This cause users to not pull correct Vagrant/Docker images, and also will affect our stats.

**Comment by remram44** [comment]
Fixed by f2484dcd.

**Comment by remram44** [comment]
```
Traceback (most recent call last):
  File "/home/remram/Documents/programming/reprozip/reprozip/reprounzip/reprounzip/parameters.py", line 57, in update_parameters
    ssl_verify=get_reprozip_ca_certificate().path)
...
  File "/home/remram/Documents/programming/_venvs/reprozip/lib/python3.6/site-packages/urllib3/contrib/pyopenssl.py", line 413, in load_verify_locations
    cafile = cafile.encode('utf-8')
AttributeError: 'bytes' object has no attribute 'encode'
```
It's hard to tell, but [git blame](https://github.com/urllib3/urllib3/blame/4239a6d9edbc617439726c7e38cdaf7b24725a29/src/urllib3/contrib/pyopenssl.py#L427) seem to indicate this was always like that. So potentially parameter fetching didn't work on PY3?

It's difficult because this goes through requests then urllib3. It might mean our stats are way under-estimated.
